user_guide_src/source/database/configuration/001.php

```diff
@@ -1,20 +1,23 @@
 <?php
 
-public $default = [
-    'DSN'      => '',
-    'hostname' => 'localhost',
-    'username' => 'root',
-    'password' => '',
-    'database' => 'database_name',
-    'DBDriver' => 'MySQLi',
-    'DBPrefix' => '',
-    'pConnect' => true,
-    'DBDebug'  => true,
-    'charset'  => 'utf8',
-    'DBCollat' => 'utf8_general_ci',
-    'swapPre'  => '',
-    'encrypt'  => false,
-    'compress' => false,
-    'strictOn' => false,
-    'failover' => [],
-];
+class Database extends Config
+{
+    public $default = [
+        'DSN'      => '',
+        'hostname' => 'localhost',
+        'username' => 'root',
+        'password' => '',
+        'database' => 'database_name',
+        'DBDriver' => 'MySQLi',
+        'DBPrefix' => '',
+        'pConnect' => true,
+        'DBDebug'  => true,
+        'charset'  => 'utf8',
+        'DBCollat' => 'utf8_general_ci',
+        'swapPre'  => '',
+        'encrypt'  => false,
+        'compress' => false,
+        'strictOn' => false,
+        'failover' => [],
+    ];
```

Comment:
There are other properties and a function.

```suggestion
    ];
    
    // ...
```